vpr/src/route/partition_tree.h

```diff
@@ -25,6 +25,19 @@ inline Side operator!(const Side& rhs) {
     return Side(!size_t(rhs));
 }
 
+/** Part of a net. Used by \ref DecompNetlistRouter. */
+class VirtualNet {
+  public:
+    /** The net in question. */
```

Comment:
Can be more precise.  I think this is the id of the original net (being decomposed) in the RouteNetlist.